src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.tsx

```diff
@@ -140,14 +273,14 @@ function ComposerWithSuggestions({
 
     const [composerHeight, setComposerHeight] = useState(0);
 
-    const textInputRef = useRef(null);
-    const insertedEmojisRef = useRef([]);
+    const textInputRef = useRef<TextInput | null>(null);
+    const insertedEmojisRef = useRef<Emoji[]>([]);
 
-    const syncSelectionWithOnChangeTextRef = useRef(null);
+    const syncSelectionWithOnChangeTextRef = useRef<SyncSelection | null>(null);
 
-    const suggestions = lodashGet(suggestionsRef, 'current.getSuggestions', () => [])();
+    const suggestions = suggestionsRef.current?.getSuggestions() ?? (() => []);
```

Comment:
```suggestion
    const suggestions = suggestionsRef.current?.getSuggestions() ?? [];
```

This looks wrong, before it was trying to execute `getSuggestions` or, if it's not defined` executing a stub function to return `[]`. So I think here we can just default to `[]`